src/Common/tests/System/Runtime/Serialization/Utils.cs

```diff
@@ -73,6 +73,7 @@ private static CompareResult CompareString(string expected, string actual)
             return new CompareResult { ErrorMessage = "actual was null, but was expecting:\n" + expected };
         }
 
+        actual = actual.Replace('\u00A0', ' ');
```

Comment:
What is the purpose of this?  Doesn't this end up changing the meaning of some tests on other platforms?

Reply:
No it will normalize spaces of serialization output. I will add the comment